lib/ui/login_page.dart

```diff
@@ -801,4 +832,83 @@ class _LoginPageState extends State<LoginPage> with WidgetsBindingObserver {
       },
     );
   }
+
```

Comment:
can we move this dialog to commons to reuse it for user inactive scenario as well